src/libs/actions/PaymentMethods.js

```diff
@@ -8,6 +9,35 @@ import Growl from '../Growl';
 import * as Localize from '../Localize';
 import Navigation from '../Navigation/Navigation';
 import * as CardUtils from '../CardUtils';
+import NameValuePair from './NameValuePair';
+
+/**
+ * Deletes a debit card
+ *
+ * @param {Number} fundID
+ *
+ * @returns {Promise}
+ */
+function deleteDebitCard(fundID) {
+    return API.DeleteFund({fundID})
+        .then((response) => {
+            if (response.jsonCode === 200) {
+                Growl.show(Localize.translateLocal('paymentsPage.deleteDebitCardSuccess'), CONST.GROWL.SUCCESS, 3000);
+                Onyx.merge(ONYXKEYS.CARD_LIST, {[fundID]: null});
+            } else {
+                Growl.show(Localize.translateLocal('common.genericErrorMessage'), CONST.GROWL.ERROR, 3000);
```

Comment:
Suggestion for follow-up: This error handling could be improved as there are probably some reasons why a fund cannot be deleted, but we tell the user nothing.